policyengine_us/parameters/gov/states/ca/tax/income/credits/earned_income/adjustment/divisor.yaml

```diff
@@ -3,7 +3,7 @@ values:
   2015-01-01: 0.85
 metadata:
   unit: /1
-  label: CalEITC adjustment factor divisor
+  label: California EITC adjustment factor divisor
```

Comment:
Could we keep it as CalEITC? The [government has adopted this shorthand](https://www.ftb.ca.gov/file/personal/credits/california-earned-income-tax-credit.html), similar to CalFresh (SNAP) and Medi-Cal (Medicaid)

Reply:
Done